rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -267,18 +290,18 @@ Apply Configuration
                     #baseUrl: https://<keycloak-url>/auth/realms/<keycloak-realm> # Uncomment for keycloak
                     #metadataUrl: .well-known/openid-configuration #Uncomment for keycloak
 
-            thirdPartyConfig:
-                flyteClient:
-                    # 3. Replace with a new Native/Public Client ID provisioned in the custom authorization server.
-                    clientId: flytectl
+                thirdPartyConfig:
+                    flyteClient:
+                        # 3. Replace with a new Native/Public Client ID provisioned in the custom authorization server.
+                        clientId: flytectl
```

Comment:
The `thirdPartyConfig` is supposed to be an element of the `appAuth` object, not on the same level.